CONTRIBUTING.md

```diff
@@ -1,32 +1,42 @@
+
+Once you located the file that you want to update, click on “Edit” which will create a new forked repo. You can make your edits there and submit a PR.
 
-Once you located the file that you want to update, click on “Edit” which will create a new forked repo. You can make your edits there, and submit a PR. Alternatively, you can manually fork the repo, create a branch in your fork and make all the edits you want before submitting a PR. You can optionally [run the application locally](https://github.com/graphprotocol/docs/blob/main/README.md) to review your changes in context.
+Alternatively, you can manually fork the repo, create a branch in your fork and make all the edits you want before submitting a PR. To view all your changes in context,[run the application locally](https://github.com/graphprotocol/docs/blob/main/README.md).
 
 ## Adding new pages
 
-When adding a new page, you should also add it to the sidebar navigation. This is done by adding an entry to the JavaScript object exported by the `_meta.js` file found in the directory of the file you created, where the key is the filename (minus the `.mdx` extension) and the value is an empty string. The `_meta.js` files control the order of the sidebar items. To give your new page a different title in the navigation (in case the full title is too long), you should add a `sidebarTitle` property to the [frontmatter](https://mdxjs.com/guides/frontmatter/) of the page, below `title`.
+When you add new page, you should also add it to the sidebar navigation. Here's how to do it:
+
+1. Add an entry to the JavaScript object exported by the `_meta.js` file found in the directory of the file you created, where the key is the filename (minus the `.mdx` extension), and the value is an empty string. The `_meta.js` files control the order of the sidebar items.
+2. To give your new page a different title in the navigation (in case the full title is too long), you should add a `sidebarTitle` property to the [frontmatter](https://mdxjs.com/guides/frontmatter/) of the page below `title`.
 
 ## Moving or renaming pages or directories
 
-If you want to make larger changes that involve moving or renaming pages or directories, you should use the `pnpm run move-pages` script in the `website` package (`cd website && pnpm run move-pages`; example usage is provided when running the script with no arguments). This will move/rename files/directories in all languages simultaneously, which is necessary to prevent build errors.
+If you want to move or rename pages or directories, you should use the following script:
+
+- Run `pnpm run move-pages` in the `website` package (`cd website && pnpm run move-pages`; example usage is provided when running the script with no arguments).
+  - This will move/rename files/directories in all languages simultaneously, which is necessary to prevent build errors.
```

Comment:
```suggestion
- `cd website`
- `pnpm run move-pages` (example usage is provided when running the script with no arguments)
- Files/directories will be moved/renamed in all languages simultaneously, which is necessary to prevent build errors.
```